### ISSUE TITLE
Interpretation of "There are 1 region(s) with p-value between 0.1 and 1e-5" message in PRSice and PRS Stability

### ISSUE THREAD

**Issue by albacamino** [choishingwan/PRSice#373]
Hello,

First of all, thank you for developing PRSice! I am new to polygenic risk score analysis, and I really appreciate any guidance you can provide.

I have a relatively small dataset with 842 individuals (338 cases and 504 controls), and I am performing k-fold cross-validation (80%-20%). Due to filtering steps (MAF, geno, etc.), I end up with 290,780 SNPs in my analysis.

I run 1,000 different folds to check the stability of my results, but I notice a lot of variation in PRS.R² values and the number of SNPs used across different runs. Additionally, I have a validation set (25 cases, 107 controls from 1000G IBS) where I compute a p-value using a t-test. However, I only get 103 significant folds (p < 0.05), about 10% of the total.

One concern I have is the message:

`There are 1 region(s) with p-value between 0.1 and 1e-5 (may not be significant).`

I am concerned about whether this message might indicate an issue with my results.

**Dataset Details:**

- Training dataset: 842 individuals (338 cases, 504 controls)
- Validation dataset: 132 individuals (25 cases, 107 controls (from IBS, 1000G))
- Filtered SNPs after QC steps (MAF, GENO, etc..): ~290,780
- Cross-validation: k-fold (80%-20%), repeated 1000 times
- PRS R² variability: High through different folds, from 0.05 to 0.2 e.g.
- Num SNPs: Num SNPs: also varies a lot in each fold, and they are not usually the same.
- Significant p-values in validation (t-test, p < 0.05): Only ~10% (103 out of 1000 folds)

**Main Questions:**

- Does the message about "1 region(s) with p-value between 0.1 and 1e-5" suggest any issue with my analysis?
- Could my PRS R² variability and low significance rate in validation be due to my parameter choices?
- Are there recommended parameter adjustments for small sample sizes to improve reproducibility?

**Command Used**
```
./PRSice_linux \
    --a1 A1 \
    --a2 A2 \
    --bar-levels 0.001,0.05,0.1,0.2,0.3,0.4,0.5,1 \
    --base folds/gwas_2_summary.txt \
    --beta  \
    --binary-target T \
    --bp BP \
    --chr CHR \
    --clump-kb 250kb \
    --clump-p 0.001000 \
    --clump-r2 0.100000 \
    --cov folds/test2_15PCA.txt \
    --interval 5e-05 \
    --ld europa.qc \
    --lower 5e-08 \
    --num-auto 22 \
    --out folds/esigen_2_PRS \
    --perm 50000 \
    --pheno folds/test2_pheno.txt \
    --print-snp  \
    --pvalue P \
    --score std \
    --seed 3791331330 \
    --snp SNP \
    --stat BETA \
    --target folds/test2_esigen \
    --thread 1 \
    --upper 0.5
```

I really appreciate any feedback or suggestions! I understand you might be busy, but I would be grateful for any insights whenever you have time to look into this.

Thank you so much! 😊

Best regards,
 Alba

**Comment by choishingwan** [comment]
It's likely a result of small sample size. When you do cross validation, did you maintain the case control ratio of each fold? If not, then it is not surprising to see such variable results considering your sample size. 

"1 region(s) with p-value between 0.1 and 1e-5" is essentially a message we print whenever there are any final PRS that has a p-values between 0.1 to 1e-5 with your phenotype. Due to large number of multiple testing, we recommend a pvalue association of 1e-5 or below for one to consider the PRS to be significant (see PRSice paper).
